extensions/shared/colorValidator.js

```diff
@@ -0,0 +1,5 @@
+const hexRegex = /^#?[A-Fa-f0-9]{6}$/;
+
+export default function colourValidator( value ) {
```

Comment:
Well if you're going to Americanize the name you should at least do it here too!

Reply:
HA!